layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteNavigationRestService.java

```diff
@@ -489,14 +503,70 @@ public Response getPages(@Context
                                                    .withSiteName(siteName)
                                                    .withDisplayName(pageDisplayName)
                                                    .build();
-      List<Page> pages = portal.findPages(pageQuery);
+      List<org.gatein.api.page.Page> pages = portal.findPages(pageQuery);
       return Response.ok().entity(pages).build();
     } catch (Exception e) {
       LOG.error("Error when retrieving pages", e);
       return Response.serverError().build();
     }
   }
 
+  @Path("/pages")
+  @POST
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @Operation(summary = "Create a page", method = "POST", description = "This creates the page")
+  @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "page created"),
+          @ApiResponse(responseCode = "400", description = "Invalid query input"),
+          @ApiResponse(responseCode = "500", description = "Internal server error") })
+  public Response createPage(@Parameter(description = "page name")
+  @QueryParam("pageName")
+  String pageName,
+                             @Parameter(description = "Page Type : GROUP, PAGE OR LINK")
+                             @QueryParam("pageTye")
+                             String pageTye,
+                             @Parameter(description = "node label")
+                             @QueryParam("link")
+                             String link,
+                             @Parameter(description = "page template : blank , normal, analytics ...")
+                             @QueryParam("pageTemplate")
+                             String pageTemplate,
+                             @Parameter(description = "site type")
+                             @QueryParam("siteType")
+                             String siteType,
+                             @Parameter(description = "site name")
+                             @QueryParam("siteName")
+                             String siteName) {
```

Comment:
```suggestion
                             @Parameter(description = "page site type")
                             @QueryParam("pageSiteType")
                             String pageSiteType,
                             @Parameter(description = "page site name")
                             @QueryParam("pageSiteName")
                             String pageSiteName) {
```
put these params just under pageName